.github/CODEOWNERS

```diff
@@ -28,4 +28,5 @@ content/en/docs/instrumentation/python/  @open-telemetry/docs-approvers @open-te
 # ruby is a content module
 content/en/docs/instrumentation/rust/  @open-telemetry/docs-approvers @open-telemetry/rust-maintainers @open-telemetry/rust-approvers
 content/en/docs/instrumentation/swift/  @open-telemetry/docs-approvers @open-telemetry/swift-mainteiners @open-telemetry/swift-approvers
+content/en/docs/k8s-operator		@open-telemetry/operator-approvers @open-telemetry/operator-maintainers
```

Comment:
note that this files is flagged with 

> This CODEOWNERS file contains errors

Reply:
should be fixed now, I added both to the list of teams having write access on this repo. cc @cartermp